### DISCUSSION TITLE
favorites - not - only from gpx-files within the folder ../favorites...!?

### DISCUSSION THREAD

**Discussion by RalfWW** [osmandapp/OsmAnd#21445] [Q&A]
Hello,

I'm trying to get my favorites, POIs, in sync.

There is a file synchronization running to a central store.
This is covert also two Android devices running OsmAnd with all gpx-files in the folder ../favorites.
The synchronization is running find, updates all changed gpx-files.

But OsmAnd is not reacting as expected.
Even closing, killing OsmAnd, re-starting the device, and starting OsmAmd
new, will keep deleted favorites. These are no longer within the synced gpx-file(s).
Where is OsmAnd getting, caching, the old content from?

How to get OsmAnd running only with the content of. /favorites/*.gpx files?

"Automatic Backup" is deactivated at both devices.

Thanks
Ralf

Device : vayu
Brand : POCO
Manufacturer : Xiaomi
Model : M2102J20SG
Product : vayu_eea
Build : SKQ1.211006.001 test-keys
Version : 12
App Version : OsmAnd+
Apk Version : 4.9.7 4907

**Comment by mariush444** [comment]
Osmand stores favorities in internal DB. If you would like to automate deletion then probably you have to use API.

**Reply by RalfWW** [reply]
Hmm, but what the gpx-files are for?

For my understanding, the files have to be the mirror of the content of a internal DB and vice versa.
Starting the app, it should load the content of the gpx files as the only knowledge base. 
It seems to me, that at the moment all loaded entries will update existing one or added as new. 
But not longer existing favorites are not removed...

That's seems to me as a "inconsequential" behavior.

Regards
Ralf

**Reply by mariush444** [reply]
it is old discussion. probably no user like it but osmand team decided this long time ago and that's all.
gpx reflects/shows/displais/ only current status of internalDB and NOT vice versa.
